thirdeye/thirdeye-pinot/src/test/java/org/apache/pinot/thirdeye/tools/RunAdhocDatabaseQueriesTool.java

```diff
@@ -576,6 +588,20 @@ private void cleanup(){
     }
   }
 
+  /**
+   * Replayed anomalies are flagged accordingly and such anomalies are excluded from the email report.
+   * This method removes the replay flag to test an email report from replayed results.
+   */
+  private void removeReplayFlagFromAnomalies() {
+    List<MergedAnomalyResultDTO> anomalies = mergedResultDAO.findByDetectionConfigAndIdGreaterThan(116257854l,0l);
```

Comment:
is this useful? can we make the id a parameter instead of hard code it?

Reply:
Added parameter.